weblab_cg/__init__.py

```diff
@@ -29,7 +29,20 @@
     load_template,
 )
 
+from ._generator_chaste_cg import (   # noqa
+    NormalChasteModel,
+    OptChasteModel,
+    Analytic_jChasteModel,
+    Numerical_jChasteModel,
+    BEOptChasteModel,
+)
+
 from ._printer import (     # noqa
```

Comment:
Should this now we from `._weblab_printer` or something? Should we have a module per output format?

Reply:
Not a bad idea, done